azure-pipelines.yml

```diff
@@ -80,7 +80,7 @@ stages:
         project: '7302639f-e704-47c3-9557-99b4e04a75bc'
         definition: '285'
         runVersion: 'latestFromBranch'
-        runBranch: "refs/heads/sic/develop"
+        runBranch: "refs/heads/sic/main"
```

Comment:
In the near future if we have cycles, we should look into defining variables for values that require changes/updates